gems/laa_multi_step_forms/app/presenters/tasks/base_task.rb

```diff
@@ -29,8 +30,12 @@ def default_url_options
     end
 
     def status
+      task_statuses.status(self.class, application:)
+    end
+
+    def current_status
       return TaskStatus::NOT_APPLICABLE if not_applicable?
-      return TaskStatus::UNREACHABLE unless can_start?
```

Comment:
So this means that if a page is reachable at any? point it should still be reachable in the future even if required pages are now not fully complete.